### DISCUSSION TITLE
Enable my app to load in an iframe in a Chrome Extension

### DISCUSSION THREAD

**Discussion by iashris** [vercel/next.js#36359] [Help]
I want to allow my NextJS app to load inside a Chrome Extension. Currently my app sends a header `X-Frame-Allow: SAMEORIGIN` which restricts this behavior, so as a work around, I am stripping it away in my extension - this feels like a hack. 

How should I disable this header altogether or allow it only for my extension? I see Next has a nifty way to disable `X-Powered-By` which is to set `poweredBy` as false in `next.config.js` - is there something similar for `X-Frame-Allow`?

**Comment by icyJoseph** [comment]
I think these two resources should help you out:

- https://nextjs.org/docs/advanced-features/security-headers#x-frame-options
- https://nextjs.org/docs/api-reference/next.config.js/headers

Basically, you can set headers for certain paths, including all, in next.config.js, and there you can set the value of `X-Frame-Allow`
